examples/mutation.go

```diff
@@ -0,0 +1,19 @@
+var mutation struct {
```

Comment:
Should I be importing packages in these files?  I just copied what was in the readme for now.

Reply:
I think this would be helpful context to have in the example - i would add it